### PR TITLE
LMSDEMO-156 - Lightbox style

### PR DESCRIPTION
**Pull request by pieecia**

### CODE REVIEW

**Comment by qunabu** [comment]
you can't import css into components, it won't work (due to webpack etc) 

you must copy css code into styled components 

see how this was done in modal https://github.com/EscolaLMS/Components/blob/main/src/components/atoms/Modal/Modal.tsx

**Review by qunabu — CHANGES_REQUESTED** [review]
you can't import css into components, it won't work (due to webpack etc)

you must copy css code into styled components

see how this was done in modal https://github.com/EscolaLMS/Components/blob/main/src/components/atoms/Modal/Modal.tsx

**Review by qunabu — CHANGES_REQUESTED** [review]
you should just import this `StyledImagePlayer` and wrap player around lightbox 

https://github.com/EscolaLMS/Components/blob/main/src/components/players/ImagePlayer/ImagePlayer.tsx

because DRY and this style is already in the code

**Comment by pieecia** [comment]
> you should just import this `StyledImagePlayer` and wrap player around lightbox
> 
> https://github.com/EscolaLMS/Components/blob/main/src/components/players/ImagePlayer/ImagePlayer.tsx
> 
> because DRY and this style is already in the code

I couldn't reuse `StyledImagePlayer` because i had a problem with interfaces, so I took style to utils and imported to components

File: src/components/molecules/MarkdownRenderer/MarkdownRenderer.tsx [hunk 1]
```diff
@@ -8,8 +8,10 @@ import rehypeRaw from "rehype-raw";
 import remarkGfm from "remark-gfm";
 import { fixContentForMarkdown } from "../../../utils/components/markdown";
 import Lightbox from "react-image-lightbox";
+import "react-image-lightbox/style.css";
```
Comment (qunabu) [inline src/components/molecules/MarkdownRenderer/MarkdownRenderer.tsx:11]:
```suggestion
```

**Review by qunabu — CHANGES_REQUESTED** [review]

**Review by qunabu — APPROVED** [review]

**Pull request merged**